modules/installation-gcp-apis.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+
+[id="installation-gcp-apis{context}"]
+= Required GCP API services
+
+To install {product-title}, you must enable the following API services in
+the service account that hosts the cluster.
+
```

Comment:
Is this a procedure, prereq, or just information?

Reply:
It renders well enough, so yes.